test/integration/claimsControllerTests.ts

```diff
@@ -0,0 +1,62 @@
+import {assert, expect} from 'chai';
+import fetch from 'node-fetch';
+import {config} from '../../src/config';
+import {performLogin} from './testUtils'
+
+describe('ClaimsControllerTests', () => {
+
+    const oauthAgentBaseUrl = `http://localhost:${config.port}${config.endpointsPrefix}`
+
+    it('Requesting claims from an untrusted origin should return a 401 response', async () => {
+
+        const response = await fetch(
+            `${oauthAgentBaseUrl}/claims`,
+            {
+                method: 'GET',
+                headers: {
+                    origin: 'https://malicious-site.com',
+                },
+            },
+        )
+
+        assert.equal(response.status, 401, 'Incorrect HTTP status')
```

Comment:
Should this be 401 or 403? I think that in the Kotlin implementation we return 403 in this case, right?